### PR TITLE
Move Banner to core

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

File: packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx [hunk 1]
```diff
@@ -16,8 +17,11 @@ import {
 # Form Field Legacy
 
 <Banner status="warning">
-  This component has been deprecated. The new [Form
-  Field](?path=/docs/documentation-lab-form-field-next--page) is a current WIP.
+  <BannerContent>
+    This component has been deprecated. The new [Form
+    Field](?path=/docs/documentation-lab-form-field-next--page) is a current
```
Comment (libertymayc) [inline packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx:22]:
```suggestion
    Field](?path=/docs/documentation-core-form-field--page) is a current
```

Comment (alycrys) [reply packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx]: that's actually the legacy one

File: packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx [hunk 2]
```diff
@@ -16,8 +17,10 @@ import {
 # Form Field Legacy
 
 <Banner status="warning">
-  This component has been deprecated. The new [Form
-  Field](?path=/docs/documentation-lab-form-field-next--page) is a current WIP.
+  <BannerContent>
+    This component has been deprecated. The new [Form
+    Field](?path=/docs/documentation-lab-form-field--page) is a current WIP.
```
Comment (libertymayc) [inline packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx:22]:
```suggestion
    Field](?path=/docs/documentation-core-form-field--page) is a current WIP.
```

I guess you checked the live storybook, but it is actually in core now if you check the sb in this PR (I don't understand when it updates on the 'live' version?)

File: packages/lab/stories/input-legacy/input-legacy.doc.stories.mdx [hunk 3]
```diff
@@ -18,8 +19,10 @@ import { withFlexGap } from "docs/decorators/withFlexGap";
 # Input Legacy
 
 <Banner status="warning">
-  This component has been deprecated. The new
-  [Input](?path=/docs/documentation-lab-input-next--page) is a current WIP.
+  <BannerContent>
+    This component has been deprecated. The new
+    [Input](?path=/docs/documentation-lab-input-next--page) is a current WIP.
```
Comment (libertymayc) [inline packages/lab/stories/input-legacy/input-legacy.doc.stories.mdx:24]:
```suggestion
    [Input](?path=/docs/documentation-core-input--page) is a current WIP.
```

**Review by libertymayc — APPROVED** [review]

File: packages/core/stories/banner/banner.doc.stories.mdx [hunk 4]
```diff
@@ -62,15 +61,15 @@ If your users have a problem that prevents them from completing a task, such as
...
 Use the warning Banner to let users know about an issue that won't prevent them from
 continuing the task, but may cause errors if it's not addressed, such as a restriction due to entitlements
 or current system status.
 
 <Canvas>
-  <Story id="lab-banner--warning" />
+  <Story id="core-banner--warning" />
 </Canvas>
 
 #### Guidance
```
[the remark this answers is not in the document]
Comment (alycrys) [reply packages/core/stories/banner/banner.doc.stories.mdx:75]: isn't this contradictory to the first comment? https://github.com/jpmorganchase/salt-ds/pull/1743#discussion_r1213139651

**Review by libertymayc — APPROVED** [review]

**Review by libertymayc — APPROVED** [review]

**Pull request merged**